### DISCUSSION TITLE
New Feature for Entity/Player Config/Movement: Player Mass

### DISCUSSION THREAD

**Discussion by Logzombie** [CapsAdmin/pac3#1125] [Ideas]
**This should not affect gravity or fall speed.**
This is a small feature but a great optional quality of life for certain large or small sized PACs in terms of scale. The player entity can be pushed by moving objects a very large amount, you could have a 14ft tall robot PAC be pushed around by a HL2 Jeep. So perhaps it were about time for a part that would allow for the player's weight value to be increased or decreased. Allowing for players to either be sent flying or simply stop said jeep in it's tracks due to how heavy the player is. Overall this is to allow for further gameplay to allow the player to not be pushed, or be pushed easier as an added detail or immersion to what somebody is trying to make. Could be fun to mess around with.

**Comment by BigBase42** [comment]
This could be an interesting feature for bigger and smaller pacs, I myself use a battle robot mech on my private server and the only way to mimic the mass/weight would be to use E2's and that in itself was hard to code for someone that hasn't used E2s ever, I've never used E2s and it took me a few hours to figure it out. This feature in its own would make the whole prosses much easier for not only me but for other users that have similar much bigger and smaller pacs.
